packages/app/src/cli/services/release.ts

```diff
@@ -81,13 +81,11 @@ export async function release(options: ReleaseOptions) {
     versionDetails.message ? `\n${versionDetails.message}` : '',
   ]
 
-  if (release.userErrors?.length > 0) {
+  if (release.userErrors) {
```

Comment:
What happens in the case of Partners if there's no error? Are we definitely not going to have a `userErrors` in the returned object? My recollection was that `userErrors` would just be an empty array, in which case release will now never work...